deploy-docker/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Deploy Docker'
+description: Deploys the build to docker
+inputs:
+  component:
+    description: 'Component name to build the image for'
```

Comment:
Can be made optional (or even removed?) since this can be automatically derived using `const { component } = getRepository()`

See e.g. `put-artefacts` action